types/ember/test/ember-tests.ts

```diff
@@ -168,3 +168,6 @@ const component1 = Ember.Component.extend(mix1, mix2, {
     lyft: Ember.inject.service(),
     cars: Ember.computed.readOnly('lyft.cars'),
 });
+
+// make sure htmlSafe returns a SafeString
+Ember.String.htmlSafe("hello").toString();
```

Comment:
This doesn't necessarily assert that `htmlSafe` returns a string. you want something like
```ts
Ember.String.htmlSafe("hello") // $ExpectType string
```
see: https://github.com/Microsoft/dtslint#write-tests

Reply:
Thanks - didn't know about those asserts. Very nice.